src/Chat/OllamaChat.php

```diff
@@ -35,15 +35,13 @@ class OllamaChat implements ChatInterface
 
     public Client $client;
 
-    private readonly LoggerInterface $logger;
-
     /** @var FunctionInfo[] */
     private array $tools = [];
 
     /** @var CalledFunction[] */
     public array $functionsCalled = [];
 
-    public function __construct(protected OllamaConfig $config, ?LoggerInterface $logger = null)
+    public function __construct(protected OllamaConfig $config, private readonly LoggerInterface $logger = new NullLogger())
```

Comment:
Shouldn't this kind of refactoring be handled by Rector once for all on all the codebase?

Reply:
https://github.com/LLPhant/LLPhant/pull/386/files#diff-d933a2ad57daa43419b483d3f1ddedaec7aa44933007fead9f9437390f596f4a